jsapp/js/components/formSummary.es6

```diff
@@ -283,15 +287,23 @@ class FormSummary extends React.Component {
       assetid: this.state.uid
     });
   }
+
   renderTeam() {
-    var team = [];
-    this.state.permissions.forEach(function(p){
-      if (p.user__username && !team.includes(p.user__username) && p.user__username != 'AnonymousUser')
-        team.push(p.user__username);
+    const team = [];
+    this.state.permissions.forEach((perm) => {
+      let username = null;
+      if (perm.user) {
```

Comment:
Was the issue that `user__username` doesn't exist? I don't see anything in `this.state.permissions` that has that attribute at this stage

Reply:
Yes that's it :)